*** linebreak.c.orig	2007-09-01 17:41:07.000000000 +0200
--- linebreak.c	2007-09-01 17:41:26.000000000 +0200
***************
*** 247,252 ****
--- 247,254 ----
      }
  }
  
+ #ifdef unused
+ 
  void
  u16_possible_linebreaks (const unsigned short *s, size_t n, const char *encoding, char *p)
  {
***************
*** 467,472 ****
--- 469,476 ----
      }
  }
  
+ #endif
+ 
  
  /* Choose the best line breaks, assuming the uc_width function.
     Return the column after the end of the string.  */
***************
*** 555,560 ****
--- 559,566 ----
    return last_column + piece_width;
  }
  
+ #ifdef unused
+ 
  int
  u16_width_linebreaks (const unsigned short *s, size_t n,
                        int width, int start_column, int at_end_columns,
***************
*** 722,727 ****
--- 728,735 ----
    return last_column + piece_width;
  }
  
+ #endif
+ 
  
  #ifdef TEST1
  
***************
*** 987,992 ****
--- 995,1002 ----
  
  #endif /* C_CTYPE_ASCII */
  
+ #if defined unused || defined TEST2
+ 
  void
  mbs_possible_linebreaks (const char *s, size_t n, const char *encoding,
                           char *p)
***************
*** 1080,1085 ****
--- 1090,1097 ----
      }
  }
  
+ #endif
+ 
  int
  mbs_width_linebreaks (const char *s, size_t n,
                        int width, int start_column, int at_end_columns,
